zebedee-reader/src/main/java/com/github/onsdigital/zebedee/search/indexing/SearchBoostTermsResolver.java

```diff
@@ -33,8 +37,10 @@ public static void loadTerms() throws IOException {
 
 
     static void loadTerms(String fileName) throws IOException {
-        InputStream resourceStream = SearchBoostTermsResolver.class.getResourceAsStream(fileName);
-        try (BufferedReader br = new BufferedReader(new InputStreamReader(resourceStream))) {
+        try (
+                InputStream resourceStream = SearchBoostTermsResolver.class.getResourceAsStream(fileName);
+                BufferedReader br = new BufferedReader(new InputStreamReader(resourceStream))
+        ) {
```

Comment:
Consider declaring the InputStreamReader in the try with resources so it is closed if the BufferedReader constructor throws an exception.